src/event_message/event_msg_builder.rs

```diff
@@ -107,9 +98,9 @@ impl EventMsgBuilder {
         EventMsgBuilder { ..self }
     }
 
-    pub fn with_delegating_seal(self, seal: LocationSeal) -> Self {
+    pub fn with_delegator(self, delegator: IdentifierPrefix) -> Self {
```

Comment:
should this take ownership on `delegator` or ref + clone? how's calling code uses it?

Reply:
Maybe this also applyes to below `SignatureThreshold` too?

Reply:
I've updated it for both. Now delegator prefix and signature threshold are passed by ref :)